src/sentry/lang/javascript/processor.py

```diff
@@ -657,6 +657,11 @@ def process_frame(self, frame, stacktrace_info, idx):
         if not frame.get('abs_path'):
             return
 
+        is_react_native = False
+        if frame['abs_path'].startswith('react-native:'):
```

Comment:
I wonder if we could change the entire path to more match `webpack://` then? I would love to have one consistent filename format for all our things. Since we get the same problem back if we look at other systems (electron etc.)